presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -2281,6 +2282,19 @@ public String getNativeExecutionExecutablePath()
         return this.nativeExecutionExecutablePath;
     }
 
+    @Config("native-execution-program-arguments")
+    @ConfigDescription("Program arguments for native engine execution")
```

Comment:
I assume the main use case for this property is to control logging levels using glog flags. It might be helpful to explain how to do that and, perhaps, link to https://rpg.ifi.uzh.ch/docs/glog.html